invenio_rdm_records/fixtures/vocabularies.py

```diff
@@ -105,20 +108,157 @@ def __iter__(self):
             yield from create_iterator(data_file)
 
 
+#
+# Exceptions
+#
+class ConflictingFixturesError(Exception):
```

Comment:
minor: usually we create an `errors.py` for this type of classes

Reply:
Perhaps let's see if another exception is needed and create the file then?